diesel/src/migration/mod.rs

```diff
@@ -53,3 +54,54 @@ impl<'a> Migration for &'a dyn Migration {
         (&**self).file_path()
     }
 }
+
+/// Create table statement for the `__diesel_schema_migrations` used
+/// used by the postgresql, sqlite and mysql backend
+pub const CREATE_MIGRATIONS_TABLE: &str = "CREATE TABLE IF NOT EXISTS __diesel_schema_migrations (\
```

Comment:
What do you think about moving this to a separate file and using `include_str!` here?